src/detect-http-headers-stub.h

```diff
@@ -324,30 +128,55 @@ static int DetectHttpHeadersSetup(DetectEngineCtx *de_ctx, Signature *s, const c
     return 0;
 }
 
+/**
+ * \brief this function setup the http.header keyword used in the rule
+ *
+ * \param de_ctx   Pointer to the Detection Engine Context
+ * \param s        Pointer to the Signature to which the current keyword belongs
+ * \param str      Should hold an empty string always
+ *
+ * \retval 0       On success
+ */
+static int DetectHttpHeadersSetupSticky(DetectEngineCtx *de_ctx, Signature *s, const char *str)
+{
```

Comment:
this needs error handling, the proper way is https://github.com/OISF/suricata/blob/master/src/detect-http-client-body.c#L159

Reply:
Will fix.